tests/bindings/test_lqubit_bindings_nb.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for LightningQubit-specific Nanobind bindings."""
+
+import numpy as np
+import pytest
+from conftest import backend
+
+# Skip all tests if not using lightning.qubit
+if backend != "qubit":
+    pytest.skip("Skipping tests for binaries other than lightning_qubit.", allow_module_level=True)
+
+
+class TestLQubitStateVectorBindings:
+    """Tests for LightningQubit-specific StateVector bindings."""
+
+    @pytest.fixture
+    def get_statevector_class(self):
+        """Get StateVector class based on precision."""
+
+        def _get_class(module, precision="64"):
+            class_name = f"StateVectorC{precision}"
+            if hasattr(module, class_name):
+                return getattr(module, class_name)
+            pytest.skip(f"Class {class_name} not available in module")
+
+        return _get_class
+
+    @pytest.mark.parametrize("precision", ["64", "128"])
+    def test_init(self, current_nanobind_module, precision, get_statevector_class):
+        """Test initialization of state vector."""
+        StateVectorClass = get_statevector_class(current_nanobind_module, precision)
```

Comment:
What about defining a fixture for the sv class w/ C64 and C128 and use that single one instead of `(self, ..., current_nanobind_module, precision, ...)` in these unit tests?